changelogs/fragments/266-default-database-for-mysql-user.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- mysql_user - added the force_context boolean option to set the default database context for the queries to be the ``mysql`` database. This way replication/binlog filters can catch the statements (https://github.com/ansible-collections/community.mysql/issues/265).
```

```suggestion
- mysql_user - added the ``force_context`` boolean option to set the default database context for the queries to be the ``mysql`` database. This way replication/binlog filters can catch the statements (https://github.com/ansible-collections/community.mysql/issues/265).
```
to highlight it a bit in the changelog